### PR TITLE
Closes #390

### LINKED ISSUE #390: Create new custom checkmate: test_range
A kind of a wrapper for:
```
test_numeric(x, finite = TRUE, any.missing = FALSE, len = 2, unique = TRUE, sorted = TRUE)
```

### PR DESCRIPTION
**Pull request by wwojciech**
Closes #390

### CODE REVIEW

File: R/checkmate.R [hunk 1]
```diff
@@ -233,3 +233,70 @@ test_length <- makeTestFunction(check_length)
...
 #' @export
 expect_length <- makeExpectationFunction(check_length)
+
+# assert_range ----
+
+#' Check that an argument is a numerical range
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' An argument `x` is a numerical range if and only if (all conditions must be met): \cr
```
Comment (danielinteractive) [inline R/checkmate.R:243]: I don't think you need all the `\cr` here and below

**Review by danielinteractive — APPROVED** [review]
very nice thanks Wojciech!

File: R/checkmate.R (hunk 1 above)
Comment (wwojciech) [reply R/checkmate.R:243]: Hmm, but it looks nice with the new lines: 
Screenshot 2022-12-21 at 12 59 34

Comment (danielinteractive) [reply R/checkmate.R:243]: I thought roxygen will handle this automatically, maybe you just need blank lines or so?

Comment (wwojciech) [reply R/checkmate.R:243]: Ohh, that is true, tx @danielinteractive!

**Pull request merged**